### PR TITLE
feat(electron): support uploading CA, client-cert, and client-key

### PR DESCRIPTION
**Pull request by XL-YiBai**
image

### CODE REVIEW

File: mqtt-client-Electron/main.css [hunk 1]
```diff
@@ -19,3 +19,7 @@
 #pathIpt {
   display: none;
 }
+
+#fileArea {
+  display: none;
+}
```
Comment (0721Betty) [inline mqtt-client-Electron/main.css:25]: css 命名格式：xxxx-xxxx

File: mqtt-client-Electron/renderer.js [hunk 2]
```diff
@@ -33,48 +33,84 @@ const subBtn = document.getElementById('subBtn')
...
+    connection
+
   let connectUrl = `${protocol.value}://${host.value}:${port.value}`
   if (protocol.value === 'ws' || protocol.value === 'wss') {
     connectUrl = connectUrl + path.value
   }
 
-  options.clientId = clientId.value || `mqttjs_${Math.random().toString(16).substr(2, 8)}`
+  options.clientId =
+    clientId.value || `mqttjs_${Math.random().toString(16).substr(2, 8)}`
```
Comment (0721Betty) [inline mqtt-client-Electron/renderer.js:113]: substr 已经被废弃了 
image

**Comment by ysfscream** [comment]
Nice work.

File: mqtt-client-Electron/main.css (hunk 1 above)
Comment (XL-YiBai) [reply mqtt-client-Electron/main.css:25]: Is the ID also the same.

Comment (ysfscream) [reply mqtt-client-Electron/main.css:25]: YES. id is the same of the class name.

File: mqtt-client-Electron/renderer.js (hunk 2 above)
Comment (ysfscream) [reply mqtt-client-Electron/renderer.js:113]: Example like:

```js
export const getClientId = () => `mqttx_${Math.random().toString(16).substring(2, 10)}` as string
```

**Review by ysfscream — APPROVED** [review]

**Pull request merged**